### PR TITLE
Milestone four: Calculations

### PR DESCRIPTION
**Pull request by krys2fa**
In this milestone, I:
- implemented the calculate.js module.
- implemented the operate.js module.
- imported the operated.js module into the calculate.js module.
- imported the calculate.js module into the App component.

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import Big from 'big';
```
Comment (Torres-ssf) [inline src/logic/operate.js]: - [ ] To import Big we need to fix this and switch to `big.js` instead

File: src/logic/calculate.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+    case '%':
+      return operate(total, next, operation);
+
+    default:
+      return 'ERROR';
+  }
+};
+
+export default calculate;
```
Comment (Torres-ssf) [inline src/logic/calculate.js]: - [ ] The calculate function is missing some of its logic. Right now is not evaluating all cases for the `buttonName`. Here we need to cover all cases for all button names (AC, =, dot button). And call `operate` only when the an operation needs to be done.

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Change required :muscle: 
Hi @krys2fa :hand: 
Great job on this milestone so far :clap: 
But we need to make some improvements before moving on :muscle: 

- [ ] Check my comments bellow

Happy Coding :computer: 
**Sergio Torres**

File: src/logic/calculate.js (hunk 2 above)
Comment (krys2fa) [reply src/logic/calculate.js]: Thanks for the review, @Torres-ssf. Changes have been effected.

**Review by fatymahmed — APPROVED** [review]
## Hi @krys2fa   👋
Good job making the requested changes 🎉🎉🎉  
## Status: Approved ✔️
_You may go ahead and merge your PR_

**Comment by krys2fa** [comment]
Thanks @fatymahmed

**Pull request merged**